### PR TITLE
Alloy Documentation - first cut

### PR DESCRIPTION
**Pull request by gs-bracej**

### CODE REVIEW

**Comment by maoo** [comment]
Pulled and tested locally; the splash screen looks great, so I'm confident to merge this PR.

File: website/siteConfig.js [hunk 1]
```diff
@@ -10,50 +10,55 @@
...
+
+const alloyHighlighter = require('./alloyHighlighter');
 
 const siteConfig = {
-  title: ' ', // Title for your website. THIS CHANGES THE TITLE TAGS FOR THE WEBSITE (LOOK IN BROWSER TAB AT TOP)
+  title: 'alloy', // Title for your website. THIS CHANGES THE TITLE TAGS FOR THE WEBSITE (LOOK IN BROWSER TAB AT TOP)
   tagline: 'Language and visual platform to generate models for the financial services industry', //THIS CHANGES THE TITLE TAGS FOR THE WEBSITE (LOOK IN BROWSER TAB AT TOP)
-  cname: 'purealloy.finos.org',
-  url: 'https://purealloy.finos.org/',
+  cname: 'alloy.finos.org',
```
Comment (maoo) [inline website/siteConfig.js]: @gs-bracej - qq. why this change? The current name is purealloy.finos.org - I've restored this value on `master` branch, please let me know if that is ok.

Comment (gs-bracej) [reply website/siteConfig.js]: Thanks! It was a snafu, caught up in the renaming from alloy -> k****s and back :)
